packages/components/CHANGELOG.md

```diff
@@ -14,6 +14,11 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### [Unreleased]
 
+#### Added
+
+- Prop `submit-button` for `Text Field Wrapper`
```

Comment:
Would be helpful to add in which context and scenario this is relevant, otherwise I might just add the prop and be surprised that nothing happens.